plugins/catalog-react/package.json

```diff
@@ -86,6 +86,7 @@
     "@backstage/cli": "workspace:^",
     "@backstage/core-app-api": "workspace:^",
     "@backstage/frontend-test-utils": "workspace:^",
+    "@backstage/plugin-catalog": "workspace:^",
```

Comment:
I know this is only a `devDependency`, but it does bring in a circular dependency here.

Is there any way you could provide a mock implementation of the `StarredEntitiesApi` instead of using the default one?

Reply:
Sure, I'll look into that 👍